omnibus/config/software/postgresql.rb

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
⏩ https://github.com/chef/omnibus-software/pull/845 Any hints on how to make this less copy-pasty are very welcome!